### PR TITLE
chore(examples): some fixes

### PR DESCRIPTION
**Pull request by Alex6323**
`process.env.MNEMONIC` was still only used in some places ... and now it's consistently used in the whole SDK.

### CODE REVIEW

File: bindings/nodejs/examples/exchange/1-create-account.ts [hunk 1]
```diff
@@ -30,8 +30,8 @@ async function run() {
                 '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
             );
         }
-        if (!process.env.MNEMONIC) {
-            throw new Error('.env MNEMONIC is undefined, see .env.example');
+        if (!process.env.NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1) {
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/exchange/1-create-account.ts]: In the exchange guide it was extra made `MNEMONIC` for https://wiki.iota.org/next/iota-sdk/how-tos/exchange-guide/#1-generate-a-mnemonic
Would look weird to use `NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1` there

Comment (Alex6323) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: ah, makes sense ... but then we need to change `.env.example` or have a separate for them, right?

Comment (Alex6323) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: tbh ... I would call it `MNEMONIC` everywhere, it's simpler, and we already say in the docs of `.env.example` that those credentials should never be used for anything serious.

Comment (Thoralf-M) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: The guide says to create one with the values

Comment (kwek20) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: yea i agree, its already weird that we have specific examples for exchange (even though theyre not so different).can at least have the code inside reused

Comment (thibault-martinez) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: I would also rename everything to `MNEMONIC`

Comment (Alex6323) [reply bindings/nodejs/examples/exchange/1-create-account.ts]: Done

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
nodejs fmt

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Comment by DaughterOfMars** [comment]
Do we really want to change the explicit name here?

**Comment by thibault-martinez** [comment]
> Do we really want to change the explicit name here?

I think it's fine tbh, we're being explicit enough with all the warnings around dotenv. And it's a bit inconsistent, if we really wanted to do that we should do it with anything related to seed/mnemonic/password everywhere is examples and we don't.

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**